src/InfoViz/Core/SelectionProvider/histogram2d.js

```diff
@@ -2,19 +2,128 @@
 // Histogram2d
 // ----------------------------------------------------------------------------
 //
+// ===> SET
+//
 //  const payload = {
 //    type: 'histogram2d',
 //    data: {
 //      x: 'temperature',
 //      y: 'pressure',
-//      bins: [...]
-//    }
+//      bins: [...],
+//      score: 0,
+//      annotationId: 24, // Generation of the annotation
+//      selectionId: 23, // Generation of the selection
+//    },
 //  }
 //
+// ===> GET
+//
 //  const query = {
 //    type: 'histogram2d',
-//    axes: ['temperature', 'pressure']
+//    axes: ['temperature', 'pressure'],
+//    score: [0, 2],
 //  }
+//
+// const response = [
```

Comment:
The form of these responses has lots of duplicated header data. What about making 'bins' an object when the query includes a score, so the response would include
bins: {"0": [...data],
           "2": [...data] },
Or bins could be an array...